### DISCUSSION TITLE
serialize(): "date" returned as member variable from __sleep() but does not exist

### DISCUSSION THREAD

**Discussion by elwafa** [spatie/laravel-queued-db-cleanup#13] [Q&A]
Hi,
While trying to use package and execute this line

Spatie\LaravelQueuedDbCleanup\CleanDatabaseJobFactory::new()
    ->query(YourModel::query()->where('created_at', '<',  now()->subMonth()))
    ->deleteChunkSize(1000)
    ->dispatch();

I got this error
` serialize(): "date" returned as member variable from __sleep() but does not exist

  at vendor/laravel/serializable-closure/src/Serializers/Signed.php:68
     64▕             throw new MissingSecretKeyException();
     65▕         }
     66▕ 
     67▕         return static::$signer->sign(
  ➜  68▕             serialize(new Native($this->closure))
     69▕         );
     70▕     }
     71▕ 
     72▕    
vendor frames 
      Laravel\SerializableClosure\Serializers\Signed::__serialize()

vendor frames 
app/Console/Commands/CleanDatabaseBigTables.php:45
      Spatie\LaravelQueuedDbCleanup\CleanDatabaseJobFactory::dispatch() `

Using PHP 8 and Laravel 8 

If anyone has any idea plz let me know 
and sorry to put this here but I can't find the Issues section in this repo

**Comment by ericclaeren** [comment]
I'm running into the same issue and haven't found a solution. Any updates on this?
